stumpy/gpu_aamp.py

```diff
@@ -439,8 +440,6 @@ def _gpu_aamp(
 
 
 def gpu_aamp(T_A, m, T_B=None, ignore_trivial=True, device_id=0, p=2.0, k=1):
-    # function needs to be revised to return (top-k) matrix profile and
```

Comment:
@NimaSarajpoor Can you also remove this from `aamped.py` as well (assuming it's already been fixed there)

Reply:
I fixed that and two other modules that have the same out-dated comment.